pkgs/applications/video/gnome-twitch/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub
+, pkgconfig, meson, cmake, ninja, wrapGAppsHook
+, libpeas, gtk3, libsoup, json-glib, webkitgtk
+, python
+, substituteAll
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gnome-twitch";
+  version = "unstable-2019-05-20";
+
+  src = fetchFromGitHub {
+    owner = "vinszent";
+    repo = "gnome-twitch";
+    rev = "8e774a04fdd15fa4c9990918bf89295a232d1715";
+    sha256 = "0pvxfwbv4fms2d0sp43jw83kp1i6gq03kg6gwy8b1fhp473gxrfj";
+  };
+
+  nativeBuildInputs = [ pkgconfig meson cmake ninja python wrapGAppsHook ];
+  buildInputs = [ libpeas gtk3 libsoup json-glib webkitgtk python ];
+
+  meta = with stdenv.lib; {
+    description = "Gnome Twitch: enjoy Twitch on your GNU/Linux desktop";
+    homepage = "http://gnome-twitch.vinszent.com";
+    license = licenses.gpl3;
```

```suggestion
    license = licenses.gpl3Plus;
```

https://github.com/vinszent/gnome-twitch/blob/master/player-backends/player-backend-gstreamer-cairo/gt-player-backend-gstreamer-cairo.c#L8